src/any_llm/providers/ollama/ollama.py

```
@@ -119,6 +119,33 @@ def _convert_list_models_response(response: Any) -> Sequence[Model]:
         """Convert Ollama list models response to OpenAI format."""
         return _convert_models_list(response)
 
+    @staticmethod
+    def _convert_response_format(
```

Ruff (0.15.20)

[warning] Too many return statements (7 > 6)

(PLR0911)